### PR TITLE
Fix hardcoded reputation bar layout; Resize according to screen

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

**Comment by yuliaveres** [comment]
not_nice
nice

File: Toshi/Controllers/Apps/Views/ReputationView.swift [hunk 1]
```diff
@@ -259,6 +259,16 @@ class ReputationBarView: UIView {
         ])
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        let barRightMargin = 11
```
Comment (Elland) [inline Toshi/Controllers/Apps/Views/ReputationView.swift]: Shouldn't this be happening through auto-layout? 🤔

Comment (yuliaveres) [inline Toshi/Controllers/Apps/Views/ReputationView.swift]: what this ? :)

Comment (Elland) [inline Toshi/Controllers/Apps/Views/ReputationView.swift]: Layout adjustments. 👍

Comment (yuliaveres) [inline Toshi/Controllers/Apps/Views/ReputationView.swift]: no, we should do this.

**Comment by Elland** [comment]
Before we merge, can you check why testing failed? This usually means CI couldn't build, since we don't have any local tests.

**Pull request merged**